### PR TITLE
fix: New installs would have multiple entries in dbSchema table

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

On new installs, entries in the DB would continuously run inserting new versions from 118 until the current. This fixes that behaviour and fixes existing installs which may be 'stuck'.

### CODE REVIEW

File: includes/sql-schema/update.php [hunk 1]
```diff
@@ -153,6 +153,9 @@
         $db_rev = $filename;
         if ($insert) {
             dbInsert(array('version' => $db_rev), 'dbSchema');
+            if ($db_rev >= 6) {
+                $insert = 0;
```
Comment (murrant) [inline includes/sql-schema/update.php]: Shouldn't this be before the $insert check?

Comment (laf) [inline includes/sql-schema/update.php]: We have to avoid the first 5 files as version doesn't exist in dbSchema until 006.sql.

Comment (murrant) [inline includes/sql-schema/update.php]: Yeah, but this code doesn't do that to my eye, it just sets $insert = 0, which is not used anywhere.

Comment (laf) [inline includes/sql-schema/update.php]: Eh, we're in a loop at this stage for each file, so we set $insert = 0 when we hit $db_rev 6 and then on the next file we don't insert anymore - we update.

**Comment by murrant** [comment]
Ah, I should have read up further.  Sorry.

**Pull request merged**